pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -113,7 +223,180 @@ def load(fnames, tag=None, inst_id=None, file_cadence=dt.timedelta(days=1),
         return data, meta
 
 
-def download(date_array, tag=None, inst_id=None, supported_tags=None,
+def load_xarray(fnames, tag='', inst_id='',
+                labels={'units': ('units', str), 'name': ('long_name', str),
+                        'notes': ('notes', str), 'desc': ('desc', str),
+                        'plot': ('plot_label', str), 'axis': ('axis', str),
+                        'scale': ('scale', str),
+                        'min_val': ('value_min', float),
+                        'max_val': ('value_max', float),
+                        'fill_val': ('fill', float)},
+                epoch_name='Epoch', meta_processor=None,
+                meta_translation=None, drop_meta_labels=None):
+    """Load NASA CDAWeb CDF files into an xarray Dataset.
+
+    Parameters
+    ----------
+    fnames : pandas.Series
+        Series of filenames
+    tag : str
+        tag or (default='')
+    inst_id : str
+        satellite id (default='')
```

Comment:
```suggestion
    tag : str
        Instrument tag (default='')
    inst_id : str
        Instrument ID (default='')
```

Reply:
What about "Data product tag" instead of "Instrument tag"?  I'm finding the difference between a tag and id confusing when written like this.

Reply:
Sure!